client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/EditorController.java

```diff
@@ -199,12 +216,7 @@ private void setCalendarColumnWidth(TableColumn column) {
     }
 
     private void setCellValueFactory(TableColumn<List<Day>, String> column, final int dayIndex) {
-        column.setCellValueFactory(new Callback<CellDataFeatures<List<Day>, String>, ObservableValue<String>>() {
-            public ObservableValue<String> call(CellDataFeatures<List<Day>, String> week) {
-                Day day = week.getValue().get(dayIndex);
-                return new SimpleStringProperty(day.toString());
-            }
-        });
+        column.setCellValueFactory(new CellValueFactoryDayCallback(dayIndex));
```

Comment:
"Could be refactored into a named static inner class"
This one is a bit confusing, here is their justification.

"This class is an inner class, but does not use its embedded reference to the object which created it.  This reference makes the instances of the class larger, and may keep the reference to the creator object alive longer than necessary.  If possible, the class should be made into a static inner class. Since anonymous inner classes cannot be marked as static, doing this will require refactoring the inner class so that it is a named inner class."

I honestly prefer the way you had it for readability, I would suppress these warnings and leave it how you had it, should I do that?